### PR TITLE
Add MetaData Files

### PR DESCRIPTION
**Pull request by acpaquette**
Adds/updates all metadata files specified in #98

Some tweaking is likely necessary in the code.json regarding the current/pending release of a 0.2.1 version of knoten

### CODE REVIEW

File: code.json [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+      "licenses": [
+        {
+          "name": "Public Domain, CC0-1.0",
+          "URL": "https://github.com/USGS-Astrogeology/knoten/blob/master/LICENSE.md"
+        }
+      ]
+    },
+
+    "homepageURL": "https://github.com/USGS-Astrogeology/knoten/releases/tag/0.1.0",
+    "downloadURL": "https://github.com/USGS-Astrogeology/knoten/archive/refs/tags/0.1.0.zip",
```
Comment (jessemapel) [inline code.json]: These URLs need to be updated for 0.2.1

File: code.json [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+    "tags": [
+      "Planetary",
+      "Remote Sensing",
+      "Photogrammetry",
+      "Testing"
+    ],
+
+    "languages": [
+      "C++"
```
Comment (jessemapel) [inline code.json]: Knoten is Python

File: code.json [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+        {
+          "name": "Public Domain, CC0-1.0",
+          "URL": "https://github.com/USGS-Astrogeology/knoten/blob/master/LICENSE.md"
+        }
+      ]
+    },
+
+    "homepageURL": "https://github.com/USGS-Astrogeology/knoten/releases/tag/0.1.0",
+    "downloadURL": "https://github.com/USGS-Astrogeology/knoten/archive/refs/tags/0.1.0.zip",
+    "disclaimerURL": "https://github.com/USGS-Astrogeology/knoten/blob/master/LICENSE.md",
```
Comment (jessemapel) [inline code.json]: This needs to be point to the new DISCLAIMER.md file and not the LICENSE.md file

File: code.json (hunk 1 above)
Comment (acpaquette) [reply code.json]: So we haven't actually released 0.2.1. I could tag a release which would make those paths valid. With no release tagged they currently go nowhere

File: code.json (hunk 3 above)
Comment (acpaquette) [reply code.json]: This needs to be handle in ISIS as well

Comment (acpaquette) [reply code.json]: PR is up for ISIS

**Review by jessemapel — APPROVED** [review]
This is good for now. We may need a second PR pointing to code.usgs.gov, but we'll do those after we mirror.

**Pull request merged**